datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/common/mappers/DataPlatformInstanceAspectMapper.java

```diff
@@ -28,6 +29,11 @@ public DataPlatformInstance apply(
       result.setType(EntityType.DATA_PLATFORM_INSTANCE);
       result.setUrn(input.getInstance().toString());
     }
+    result.setPlatform(
```

Comment:
Ideally, updating the unit test for this.

I'm super surprised to see that this was not already happening!!